src/dagmc/DagMC.hpp

```diff
@@ -466,6 +466,56 @@ class DagMC {
     return MBI_shared_ptr;
   }
 
+  // Verbosity levels
+  // -1 : "override" the current verbosity setting and write message
+  //  0 : No output
+  //  1 : All output
+  int set_verbosity(int val) {
+    int verbosity_max = 0;
+    int verbosity_min = 1;
+    if (val < verbosity_min || val > verbosity_max)
+      warning("Invalid verbosity value " + std::to_string(val) +
+              " will be set to nearest valid value.");
+    val = std::min(std::max(verbosity_min, val), verbosity_max);
+    return verbosity = val;
```

Comment:
Unless my brain is missing something, this will always result in 0?

Reply:
Mm yeah mentally bit-flipped the initializer values at the beginning of this function.